baseline/pytorch/tagger/model.py

```diff
@@ -463,8 +471,7 @@ def compute_loss(self, inputs):
         tagging_loss = self.decoder.neg_log_loss(unaries, tags, lengths)
 
         #classification loss
-        B = embed.shape[0]
-        embed = embed[:,0,:].reshape(B,-1)
+        embed = self.pool_model(embed)
         class_out = self.proj_layer_classification(embed)
         class_loss = nn.NLLLoss()(class_out, class_labels)
```

Comment:
IF you are going to instantiate the object every time, use `torch.nn.functional` version instead.